### ISSUE TITLE
Generic drug not showing in the 'Select prescription drug(s)' page, but its brand name is showing on this page. 

### ISSUE THREAD

**Issue by MCIR** [christillman/encounterpro_os#75]
The medication is authorized in KE, UG. The brand name 'Ascoril SF' is showing in the 'Select prescription drug(s)' page, but the generic selection 'bromhexine/guaifenesin/terbutaline' is not showing. Could this be because this generic drug is not configured to show?
![image](https://github.com/user-attachments/assets/ac7d059d-b97b-472e-b893-7c3ed06a68b9)

**Comment by christillman** [comment]
Hi Ciru,

I couldn't replicate this one. Could you check again, perhaps the script you executed for #74 could have fixed it (because it involved the same Ascoril medication).

**Comment by MCIR** [comment]
Hi Chris, 

 

The script executed for #74 has not resolved the issue. 



 

The combination generic drug 'bromhexine/guaifenesin/terbutaline' does not appear in search results at all, even when a search is made using the individual generic terms 'bromhexine', 'terbutaline', or 'guaifenesin'. However, brand-name drugs containing this combination do appear in the search results when searched by their brand names. For instance, ASCORL SF as seen in the screenshot above.

**Comment by christillman** [comment]
Hi Ciru,

I have it showing up when I search Generic Name for bromhex, but only if I use All instead of specialty.

![image](https://github.com/user-attachments/assets/b937a4b3-fc6e-40b1-8f67-cb1bc9e2cd2e)

**Comment by christillman** [comment]
Oh, I think I see. You can pick the generic inside Ascoril, but the generic is not listed at the left to pick first.

![image](https://github.com/user-attachments/assets/bd397504-4218-4cb7-b240-f829b8a036d4)

**Comment by MCIR** [comment]
Hi Chris, 

 

Yes. The brand drug name Ascoril SF will show then in brackets it shows that
it contains guaifenesin, bromhexine, terbutaline. 

However, if you search for the generic drug guaifenesin / bromhexine /
terbutaline no results are showing

 



 

The search results should show 2 drugs. The generic drug guaifenesin /
bromhexine / terbutaline and the second should be Ascoril SF. Ascoril is the
only brand name drug that contains guaifenesin / bromhexine / terbutaline.
See below. The reason for following this up is, this error could be
affecting many other drugs, but it is difficult to pick up which ones might
be affected.

**Comment by christillman** [comment]
Indeed, there are a few others in the same boat. I think the 2 Ascoril ones were related to the naming problem that caused missing first and last letters in the generic ingredients.

The vaccines can probably be deleted, they came from RXNorm and I'm sure are no longer applicable.

I'll fix the data for the Ascoril and Sediproct generics in the next release.

drug_id	drug_type	common_name	generic_name
RXNB2178358	Vaccine	Fluad 2019-2020	influenza A virus (H1N1) antigen / influenza A virus (H3N2) antigen / influenza B virus antigen
RXNB2177693	Vaccine	Fluzone 2019-2020	influenza A virus (H1N1) antigen / influenza A virus (H3N2) antigen / influenza B virus antigen
RXNB2381146	Vaccine	Fluad 2020-2021	influenza A virus (H1N1) antigen / influenza A virus (H3N2) antigen / influenza B virus antigen
RXNB1801607	Vaccine	Flucelvax Quadrivalent 2016-2017	influenza A virus A/Brisbane/10/2010 (H1N1) antigen / influenza A virus A/Hong Kong/4801/2014 (H3N2) antigen / influenza B virus B/Hong Kong/259/2010 antigen / influenza B virus B/Utah/9/2014 antigen
RXNB1801825	Vaccine	Flumist Quadrivalent 2016-2017	influenza A virus A/Bolivia/559/2013 (H1N1) antigen / influenza A virus A/New Caledonia/71/2014 (H3N2) antigen / influenza B virus B/Brisbane/60/2008 antigen / influenza B virus B/Phuket/3073/2013 antigen
RXNB1946970	Vaccine	Flumist Quadrivalent 2017-2018	influenza A virus A/New Caledonia/71/2014 (H3N2) antigen / influenza A virus A/Slovenia/2903/2015 (H1N1) antigen / influenza B virus B/Brisbane/60/2008 antigen / influenza B virus B/Phuket/3073/2013 antigen
UGBI2136	Single Drug	SEDIPROCT SUPOSITORY	No longer available on 11th May
UGBI6424	Single Drug	Ascoril LS	ambroxol HCL / levosalbutamol / guaifenesin
UGBI6489	Single Drug	Ascoril SF	bromhexine / guaifenesin / terbutaline

**Comment by MCIR** [comment]
Ok thanks. You are right about the vaccines.
